packages/react-core/src/components/Toolbar/ToolbarGroup.tsx

```diff
@@ -166,6 +166,15 @@ export interface ToolbarGroupProps extends Omit<React.HTMLProps<HTMLDivElement>,
     xl?: 'wrap' | 'nowrap';
     '2xl'?: 'wrap' | 'nowrap';
   };
+  /** Flex grow modifier at various breakpoints */
```

Comment:
```suggestion
  /** Indicates whether a flex grow modifier of 1 is applied at various breakpoints */
```
Small nit to the description, just to make it clear that it's currently a hardcoded flex grow modifier rather than something more customizable.